h/static/scripts/widget-controller.js

```diff
@@ -72,6 +94,9 @@ module.exports = function WidgetController(
   var visibleThreads = new VirtualThreadList($scope, window, thread());
   annotationUI.subscribe(function () {
     visibleThreads.setRootThread(thread());
+    $scope.totalAnnotations = countAnnotations(annotationUI.getState().annotations);
+    $scope.totalNotes = countNotes(annotationUI.getState().annotations);
+    $scope.tabSelection = annotationUI.getState().selectedTab;
```

Comment:
It confused me what `tabSelection` was when looking through the code above. I would suggest naming it `selectedTab` for consistency.